src/sql/workbench/browser/modelComponents/dropdown.component.ts

```diff
@@ -35,6 +38,10 @@ import { ILogService } from 'vs/platform/log/common/log';
 		</div>
 		<div [style.display]="getEditableDisplay()" #editableDropDown style="width: 100%;"></div>
 		<div [style.display]="getNotEditableDisplay()" #dropDown style="width: 100%;"></div>
+		<div class="dropdown-error-container" #errorMessage *ngIf="!isValid && validationErrorMessage.length!==0 && !isInitState">
```

Comment:
is this approved by the UX team?

Reply:
I was using this design pattern from the portal. Do I still need to get the UX approval?